src/utils/address/startsWithOrEndsWith.js

```diff
@@ -0,0 +1,12 @@
+// @flow strict
+
+import { endsWith } from 'lodash-es'
+
+export function startsWithOrEndsWith(initialAddress: string, initialQuery: string): boolean {
+  const address = initialAddress.toLowerCase()
+  const query = initialQuery.toLowerCase()
+
+  return address.startsWith(query)
+    || address.startsWith(`0x${query}`)
```

Comment:
isn't it possible to pass query, that already contain `0x`?
probably, need to use `add0x` util function here

Reply:
`.startsWith` [is not works](http://kangax.github.io/compat-table/es6/#test-String.prototype.startsWith) in Internet Explorer

Reply:
Does babel transform it for IE?